meilisearch-http/src/routes/api_key.rs

```diff
@@ -29,8 +29,8 @@ pub async fn create_api_key(
     body: web::Json<Value>,
     _req: HttpRequest,
 ) -> Result<HttpResponse, ResponseError> {
-    let key = auth_controller.create_key(body.into_inner()).await?;
```

Comment:
No!! this function in blocking, you can't call it in this context! You need to make them async again, and call the store operation from a `tokio::spawn_blocking` context